packages/react-scripts/scripts/utils/createJestConfig.js

```diff
@@ -41,6 +41,8 @@ module.exports = (resolve, rootDir, isEjecting) => {
         : resolve('config/jest/babelTransform.js'),
       '^.+\\.css$': resolve('config/jest/cssTransform.js'),
       '^(?!.*\\.(js|jsx|css|json)$)': resolve('config/jest/fileTransform.js'),
+      // ZEAL: Adds support for parsing GraphQL files
+      '^.+\\.(gql|graphql)$': require.resolve('jest-transform-graphql'),
```

Comment:
I started with `resolve`, but it doesn't work here. They're using it to load local config files that may be in a different place after you eject. In each of those files they're using `require.resolve` for this sort of thing.